pkg/cmd/mom/apply_configuration_command.go

```diff
@@ -0,0 +1,35 @@
+package mom
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openshift/cluster-authentication-operator/pkg/operator"
+
+	"github.com/openshift/multi-operator-manager/pkg/library/libraryapplyconfiguration"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+)
+
+func NewApplyConfigurationCommand(streams genericiooptions.IOStreams) *cobra.Command {
+	return libraryapplyconfiguration.NewApplyConfigurationCommand(RunApplyConfiguration, streams)
+}
+
+func RunApplyConfiguration(ctx context.Context, input libraryapplyconfiguration.ApplyConfigurationInput) (libraryapplyconfiguration.AllDesiredMutationsGetter, error) {
```

Comment:
have you considered returning a type that would implement `OperatorStarter`? If we could do that, we would have more control over `RunOnce`.

Reply:
> have you considered returning a type that would implement `OperatorStarter`? If we could do that, we would have more control over `RunOnce`.

doing this in https://github.com/openshift/multi-operator-manager/pull/12 and will revendor